packpack

```diff
@@ -18,6 +18,25 @@ DOCKER_REPO=${DOCKER_REPO:-packpack/packpack}
 # Extra parameters for docker run.
 PACKPACK_EXTRA_DOCKER_RUN_PARAMS="${PACKPACK_EXTRA_DOCKER_RUN_PARAMS:-}"
 
+# Set CI variable to CI provider name
+CI="false"
+if [ "${APPVEYOR}" = "True" -o "${APPVEYOR}" = "true" ]; then
```

Comment:
I would use `:-` (default value) construct like so: `"${APPVEYOR:-}"` — in order to keep the script working with `set -u` (it prevents expanding of an unset variable). It is the rule of thumb for shell scripts.

Reply:
Corrected.